src/bmp.c

```diff
@@ -46,3 +45,54 @@ void write_bmp(char *filename, char rgb[], int length, int width) {
     fclose(fp);
     free(bitmap);
 }
+
+// Define portion of the set to be displayed. The more the decimals, the more inside in.
+const double set_left = -2.5;
+const double set_right = 1.5;
+const double set_down = -2;
+const double set_up = 2;
+
+void ismandelbrot(int world_x, int world_y, char *rgb){    

```

Comment:
This function should be declared in its header file (maybe fractal.h?) and implemented in its corresponding .c file.